bundle/Resources/views/default/content/text_linked.html.twig

```diff
@@ -2,7 +2,7 @@
 {# location \Netgen\IbexaSiteApi\API\Values\Location #}
 
 {% if location is defined %}
-    <a href="{{ ibexa_path(location) }}">{{ content.name }}</a>
+    <a href="{{ location.url }}">{{ content.name }}</a>
 {% else %}
-    <a href="{{ ibexa_path(content) }}">{{ content.name }}</a>
+    <a href="{{ content.url }}">{{ content.name }}</a>
```

Comment:
Here too, in two locations should be `content.path` and `location.path`

Reply:
Updated in https://github.com/netgen/ibexa-site-api/pull/11/commits/c734f615167a5ab8333947c44ff8c2fd41448bdb